src/SpecFlow.NetCore/Fixer.cs

```diff
@@ -65,13 +53,32 @@ private string FindSpecFlow(string path)
 			throw new Exception($"Can't find SpecFlow: {path}\nTry specifying the path with {Args.SpecFlowPathArgName}.");
 		}
 
+		private static bool TryGetSpecFlowVersion(FileInfo csproj, out string version)

```

Comment:
Again, reason for `static`?

Reply:
It's like a hint (for those who will read this code) that the method does not use class members.